apps/frontend/src/app/2_molecules/SystemStats/components/ZeroStats.tsx

```diff
@@ -108,11 +112,32 @@ export const ZeroStats: FC<ZeroStatsProps> = ({ className, dataAttribute }) => {
     [zusdInStabilityPool, zeroInStabilityPoolPercent],
   );
 
+  const renderBTCPrice = useMemo(
+    () =>
+      btcPrice ? (
+        <AmountRenderer
+          value={btcPrice}
+          suffix={USD}
+          precision={USD_DISPLAY_PRECISION}
+          showRoundingPrefix={false}
+          dataAttribute="zero-statistics-zusd-btc-price"
+        />
+      ) : (
+        0
+      ),
+    [btcPrice],
+  );
+
   const renderProtocolMode = useMemo(
     () => (isRecoveryMode ? SystemModeType.RECOVERY : SystemModeType.NORMAL),
     [isRecoveryMode],
   );
 
+  const originationFeeRate = useMemo(
+    () => formatValue(decimalic(minBorrowingFeeRate).mul(100), 2),
```

Comment:
`minBorrowingFeeRate` is already decimalic

```suggestion
    () => formatValue(minBorrowingFeeRate.mul(100), 2),
```